scripts/main/lychee_locale.js

```diff
@@ -130,6 +130,7 @@ lychee.locale = {
 	STARRED: "Starred",
 	RECENT: "Recent",
 	PUBLIC: "Public",
+	ON_THIS_DAY: "On This Day",
```

Comment:
You will want to modify also the following files:
https://github.com/LycheeOrg/Lychee/tree/master/app/Locale :)

Reply:
Thanks, I missed that. I'm going to update those files with the help of google translate :)

Reply:
You don't necessarily need to use Google translate, you will notice that there are some untranslated strings still in English left in each languages. :)

Reply:
You're right. It's safer to leave a native speaker to do a translation from English :)